.eslintrc.json

```diff
@@ -1,3 +1,15 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": ["next", "next/core-web-vitals", "eslint:recommended"],
+  "globals": {
+    "React": "readonly"
```

Comment:
Because we we don't necessarily need to import React into the scope of the components after react 17. This tells eslint if we don't import React, it's ok and there's no need to throw a warning for this.